Source/WebCore/css/process-css-properties.py

```diff
@@ -2663,12 +2653,6 @@ def generate_css_property_names_gperf(self):
                 iterable=self.properties_and_descriptors.all_descriptor_only
             )
 
-            self.generation_context.generate_property_id_switch_function_bool(
-                to=writer,
-                signature="bool CSSProperty::shouldPreserveWhitespace(CSSPropertyID id)",
```

Comment:
You should get rid of the declaration in `CSSProperty.h` as well.

Reply:
Good point.